packages/sit-onyx/src/components/OnyxDataGrid/features/resizing/resizing.ts

```diff
@@ -34,16 +40,32 @@ export const useResizing = createFeature(
       { flush: "post", deep: true },
     );
 
+    window.addEventListener("resize", () => {
```

Comment:
I'd rather prefer to watch the table itself for resizes since it can e.g. also be resized by CSS / grid settings without the actual window being resized.

We have a `useResizeObserver()` utiltiy that we can use to observe the table :)